stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Why is this file called ...overlay.yaml? Is this an overlay? Should it be in the overlay folder then?

Reply:
Sorry about that, the monitoring overlay I created accidentally got added to this pull request as well. I will move it to the correct folder. Other than that does it look alright?

Reply:
yes, thank you!

Reply:
Should be good now. I noticed charmed kubernetes uses `monitoring-pgt-overlay.yaml` so I named it `openstack-base-monitoring-pgt-overlay.yaml`.